recipes/ayufan-release-contents.md

```diff
@@ -0,0 +1,151 @@
+# Ayufan release contents
+
+Ayufan linux images are built specifically for the `rock64`.
+
+  * The latest [stable release](https://github.com/ayufan-rock64/linux-build/releases/latest) is `0.5.15`
+  * The latest [experimental release](https://github.com/ayufan-rock64/linux-build/releases) is `0.6.x` (updated often)
+
+It is **not recommended** to install an **experimental release**, except to obtain certain tools such as the `u-boot-flash` image.
+
+This document aims to answer the questions:
+
+  1. [What](#what-is-included) is included in these releases?
+  2. [How](#how-are-they-built) are these releases built?
+  3. [Can](#can-they-be-trusted) these releases be trusted?
+  4. [But](#but-i-want-my-own) can I modify these releases?
+  5. [Who](#who-is-ayufan) is _ayufan_?
+  6. [Are](#are-there-alternatives) there alternatives to these releases?
+
+## What is included
+
+Every build contains the following files available for download:
+
+  * Popular Linux distributions (ex: Debian, Ubuntu, Container Linux)
+  * Linux kernel and headers
+  * Debian package with custom tools, wallpapers, start scripts, etc
+  * U-boot flash and erase images (since `0.6.10`)
+  * Source code for all build scripts
+
+Let's focus on the Debian package `linux-rock64-package-0.x.x_all.deb`, whose contents can be [found here](https://github.com/ayufan-rock64/linux-build/tree/master/package/root).
+
+This package is installed on all builds. It's essentially the _"custom"_ stuff added to each base OS.
+
+```
+# linux-rock64 package contents
```

Comment:
Can we instead of doing that show the linux-build tree where all the files are defined?